dataflow/gpu-examples/pytorch-minimal/README.md

```diff
@@ -0,0 +1,39 @@
+# PyTorch GPU minimal pipeline
+
+## Before you begin
+
+Make sure you have followed the
+[Dataflow setup instructions](../../README.md).
+
+## Building the Docker image
+
+We use
+[Cloud Build](https://cloud.google.com/build)
+to build the container image for the workers and save it in
+[Container Registry](https://cloud.google.com/container-registry/).
+
+```sh
+gcloud builds submit --config build.yaml
+```
+
+## Running the Dataflow job with GPUs
+
+We use Cloud Build to run the [Dataflow](https://cloud.google.com/dataflow) job.
+
+> ℹ️ We launch the job using the worker image to make sure the job launches
+> with the same Python version as the workers and all the dependencies installed.
+
+```sh
+export REGION="us-central1"
+export GPU_TYPE="nvidia-tesla-t4"
+
+gcloud beta builds submit \
```

Comment:
If it's not necessary to use beta here, let's use regular `gcloud builds submit`

Reply:
I was using beta because only beta supports streaming the logs, so without beta it looked like a black box process that hanged indefinitely. It definitely works without the beta. There is a console message suggesting to use beta if you want logs, so I guess we can leave that to the user.